### PR TITLE
Updated the ui/pages/performance to Sistent and MUI v5

### PR DESCRIPTION
**Pull request by Ajay-singh1**
**Notes for Reviewers**

I have updated the ui/pages/performance to Sistent and MUI v5 based on the sheet provided in Google Docs

Screenshot from 2024-11-18 08-36-30

Screenshot from 2024-11-18 08-37-59

- This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ajay-singh1** [comment]
@aabidsofi19 Can you kindly review this PR?

**Comment by vishalvivekm** [comment]
@Ajay-singh1 
Let's discuss this during the Meshery Development call today at 9:00 AM CT | 8:30 PM IST. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#) || [meeting link](https://meet.layer5.io/meshery).

File: ui/pages/performance/results.js [hunk 1]
```diff
@@ -1,15 +1,14 @@
 import React, { useEffect } from 'react';
-import { NoSsr, Paper, withStyles } from '@material-ui/core';
+import { NoSsr } from '@mui/material';
+import { Paper, SistentThemeProvider } from '@layer5/sistent';
```
Comment (sudhanshutech) [inline ui/pages/performance/results.js:3]: not required to use Sistentthemeprovider. There is already a wrapper use that : usesSisistent

Comment (sudhanshutech) [reply ui/pages/performance/results.js:3]: and why we directly updating pages, are the parent component updated to sistent that is here MesheryResults? That should done first.

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
@Ajay-singh1 is this still a draft or is it ready for review?

**Comment by Ajay-singh1** [comment]
@leecalcote It is still in draft!

**Pull request closed without merging**